packages/notifi-core/lib/models/FusionEventDescriptor.ts

```diff
@@ -0,0 +1,5 @@
+export type FusionEventDescriptor = {
```

Comment:
I think we can just simply use the auto generated type `Types.FusionEventDescriptor`, cant we?